fields.yml

```diff
@@ -273,14 +267,22 @@
         - name: domain
           level: core
           type: keyword
+          example: www.example.com
           description: >
-            Destination domain.
+            Destination domain in full.
     
-        - name: subdomain
-          level: core
+        - name: registered_domain
+          level: extended
           type: keyword
+          example: example.com
           description: >
-            Destination subdomain.
+            Destination domain, with the subdomain stripped out.
```

Comment:
Perhaps we could also include that this is the “effective top level domain plus one additional level” as secondary description to remove any ambiguity.

Reply:
Done